### PR TITLE
RUN-113: unable to delete project with 100 k executions

### PR DESCRIPTION
**Pull request by alexander-variacode**
Issue:
https://github.com/rundeckpro/rundeckpro/issues/1240

Fix description:
Modified the project deletion action to improve the process and also it is added a spinner with a description to show the deletion process is running.

Screen Shot 2021-07-08 at 18 31 15

### CODE REVIEW

**Comment by ronaveva** [comment]
@alexander-variacode could you upload a project with such a big number of executions to give it a test?
also, it looks like "build_and_test" task got stuck

File: rundeckapp/grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -44,6 +44,7 @@ default.paginate.prev=-
 default.paginate.next=+
 default.paginate.rew=«
 default.paginate.ff=»
+action.take.amount.time=This action could take a significant amount of time
```
Comment (ronaveva) [inline rundeckapp/grails-app/i18n/messages.properties:47]: I think this is a more appropriate message
"This action might take a while"

**Review by ronaveva — CHANGES_REQUESTED** [review]
Overall looks good. But rebase is needed

File: rundeckapp/grails-app/i18n/messages.properties (hunk 1 above)
Comment (alexander-variacode) [reply rundeckapp/grails-app/i18n/messages.properties]: Done...

**Comment by alexander-variacode** [comment]
@Mergifyio rebase

**Review by ronaveva — APPROVED** [review]
LGTM

**Pull request merged**